mixer/adapter/signalfx/operatorconfig/tracing.yaml

```diff
@@ -0,0 +1,58 @@
+---
+apiVersion: config.istio.io/v1alpha2
+kind: tracespan
+metadata:
+  name: signalfx
+  namespace: istio-system
+spec:
+  traceId: request.headers["x-b3-traceid"] | ""
+  spanId: request.headers["x-b3-spanid"] | ""
+  parentSpanId: request.headers["x-b3-parentspanid"] | ""
+  spanName: request.path | "/"
```

Comment:
just an FYI: this will include full query string.

Reply:
Hmm...ok, I think I will add some logic to parse out query strings and add them as tags.  Unless there is another value that would be better to use.  I tried `api.operation` but it is blank for the bookinfo sample app and so probably not very reliable as a general purpose span name.

Reply:
there is an open issue for providing a better way to distinguish `path` from `queryParams`, so we might have a better story in a little bit, but nothing at the moment.